src/OpenTelemetry.Api/Trace/SpanAttributeConstants.cs

```diff
@@ -27,5 +27,9 @@ internal static class SpanAttributeConstants
         public static readonly string HttpRequestSizeKey = "http.request.size";
         public static readonly string HttpResponseSizeKey = "http.response.size";
         public static readonly string HttpRouteKey = "http.route";
+
+        public static readonly string ErrorNameKey = "error.name";
```

Comment:
are error conventions documented in the spec? I cannot find anything there

Reply:
I don't think so - but it's a very normal .net convention; and it's supported StackDriver at least